app/bundles/CampaignBundle/Assets/js/campaign.js

```diff
@@ -1725,6 +1764,9 @@ Mautic.campaignBuilderUpdateEventList = function (groups, hidden, view, active,
         }
     });
 
+    mQuery('#CampaignEventPanelGroups').removeClass('groups-enabled-1 groups-enabled-2 groups-enabled-3')
+    mQuery('#CampaignEventPanelGroups').addClass('groups-enabled-' + groups.length)
```

Comment:
Can you explain what this is for? I can see the `groups-enabled-2` has some CSS rule, but nothing else. What happens with `groups-enabled-4`?

Reply:
It's the number of available event types to select in this window:
![image](https://github.com/mautic/mautic/assets/8580942/4b19e097-ea57-45e5-b2b0-4da80bbe3884)
For example, after the decision event, you can only choose action or condition, so the window will have the class `groups-enabled-2` so that it can be properly styled for that size. The 3 groups are max at this moment, I'm clearing here all possible group classes before adding the current one.